ynput/ayon-core
# Fix Ayon Load Media PR logic and refactor to latest develop

## Changelog Description

- Updated to latest develop
- Fixed some refactoring to  `ayon_resolve`
- Fixed remove unused media inventory action + improved label
- Did a test run

## Additional info

I couldn't push into the upstream branch - so am just doing it this way.

- [x] Tested loading and managing Load Media loader with both H264 and EXR.
- [x] Tested loading and managing Load Clip loader with both H264 and EXR
- [x] Tested Workfiles tool
- [x] For publishing I only tested just publishing workfile (no other instances) since I'm not sure how that legacy creator works.

## Testing notes:

1. Run resolve
2. Load and publish should all work.
